app/assets/stylesheets/answers.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Answers controller here.
```

Comment:
At some point moving to inline JS will be a good exercise, but I think you Sass is best until we get React in place. Its easier to get up and running this way. I'm open to talking about it, but I haven't had time to look into Radium, etc... might be better to get going and think about inline later. @muffy - thoughts?